### PR TITLE
execute mdx api changes

### PR DESCRIPTION
**Pull request by labkey-matthewb**
Same as 19.3 version, just going into develop instead

### CODE REVIEW

File: api/src/org/labkey/api/view/ViewServlet.java [hunk 1]
```diff
@@ -576,13 +578,26 @@ public static MockHttpServletResponse mockDispatch(HttpServletRequest request, @
 
         MockHttpServletResponse mockResponse = new MockHttpServletResponse()
         {
+            {
+                setCharacterEncoding(StringUtilsLabKey.DEFAULT_CHARSET.name());
+            }
```
Comment (labkey-susanh) [inline api/src/org/labkey/api/view/ViewServlet.java:583]: Why curly braces here?

File: query/src/org/labkey/query/olap/rolap/RolapTestCase.java [hunk 2]
```diff
@@ -925,9 +923,9 @@ public void testRegressionsUnion() throws Exception
             stmt = conn.createStatement();
             assertNotNull(stmt);
             QueryProfiler.getInstance().ensureListenerEnvironment();
-            long ms = System.currentTimeMillis();
+            //long ms = System.currentTimeMillis();
             cs = stmt.executeOlapQuery(mdx);
-            long d = System.currentTimeMillis() - ms;
+            //long d = System.currentTimeMillis() - ms;
```
Comment (labkey-susanh) [inline query/src/org/labkey/query/olap/rolap/RolapTestCase.java:928]: Are these commented lines still desired?

**Review by labkey-susanh — APPROVED** [review]
Just a couple of comments noted in passing.

File: api/src/org/labkey/api/view/ViewServlet.java (hunk 1 above)
Comment (labkey-matthewb) [reply api/src/org/labkey/api/view/ViewServlet.java:583]: That's the constructor for the anonymous class.

**Pull request merged**